content/docs/rfcs/23/README.md

```diff
@@ -145,17 +145,19 @@ For mapping Waku v1 topics to Waku v2 content topics,
 the following structure for the content topic SHOULD be used:
 
 ```
-/waku/1/<4bytes-waku-v1-topic>/rlp
+/waku/1/<4bytes-waku-v1-topic>/rfc7
```

Comment:
This should probably be https://rfc.vac.dev/spec/26/ as it is more complete

> This specification effectively replaces 7/WAKU-DATA as well as 6/WAKU1 Payload encryption but written in a way that is agnostic and self-contained for Waku v1 and Waku v2.

Reply:
Right! Should have remembered that. Changed to `/rfc26` in f942a81